doc/source/whatsnew/v0.20.0.txt

```diff
@@ -41,6 +41,13 @@ Other enhancements
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
 
+- New ``UnsortedIndexError`` (subclass of ``KeyError``) thrown when indexing/slicing into an
+  unsorted MultiIndex (:issue:`11897`). This allows differentiation between errors due to lack
+  of sorting or an incorrect key. See :ref:`here <advanced.unsorted>`
+ 
+- Change error message text corresponding to error raised when indexing via a
+  boolean ``Series`` that has an incompatible index (:issue:`14491`)
```

Comment:
I shortened the item.  I think it should be in whatsnew to refer back to the issue.  But you guys can decide!